misc/testerina/modules/testerina-core/src/main/ballerina/src/test/Module.md

```diff
@@ -0,0 +1,25 @@
+## Module overview
+
+This module allow developers to write testable code.
```

Comment:
```suggestion
This module allows developers to write testable code.
```

Reply:
Not exactly what it does right? What this module provides is a framework for developers to write tests for their code. This module does not make what the user writes testable.